test/projects/test_lps.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from runner import ProjectType
+from glotter import project_test, project_fixture
+from test.utilities import clean_list
+
+invalid_permutations = (
+    'description,in_params,expected', [
+        (
+            'no input',
+            None,
+            'Incorrect input provided. Program Terminated'
+        ), (
+            'empty input',
+            "",
+            'Incorrect input provided. Program Terminated'
+        ), (
+            'invalid input: no palindromic present',
+            "Polip",
+            'No Palindromic substring present.'
+        ),
+        (
+            'invalid input: no palindromic present',
+            "Polip",
+            'No Palindromic substring present.'
+        )
+    ]
+)
+
+valid_permutations = (
+    'description,in_params,expected', [
+        (
+            'sample input: longest palindrome of 6eie6',
+            '6eie6o6',
+            'Longest Palindromic Substring is: 6eie6'
```

Comment:
This is great! And, it actually replicates the table perfectly. That said, I think we need to revisit the documentation to make the script a bit simpler. I think for the valid cases we should return the LPS directly. Then, we should duplicate the error string for all error cases. Let me take a crack at that in the next day or so.

Reply:
Wouldn't the output need to be changed also in all the files for this problem to match the output on the test cases in order to pass the test or am I misunderstanding how these tests work?

Reply:
No, you're completely right! Usually, the challenge with adding these tests is that any existing scripts will need to be fixed. That said, we're in a phase of addressing technical debt at the moment, so it's better to do these things now than later. Ultimately, we're hoping to get all testing added to the repo before we allow new solutions.